### PR TITLE
Implemented Equipment and Cosmetic Slots

### PR DESCRIPTION
**Pull request by bwu113**
Database functions for equipping and unequipping regular equipment and cosmetics.
Updated all tablemanager local variables and parameters to comply with camelCase.

### CODE REVIEW

File: Ms2Database/Controllers/EquipmentManager.cs [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+                                             .FirstOrDefault(column => column.EquipmentSlot == slot.ToUpper());
+
+                if (equippedItem == null || equippedItem.EquipmentSlot == "")
+                {
+                    item.EquipmentSlot = slot.ToUpper();
+                }
+                else
+                {
+                    UnequipItem(equippedItem.UniqueId, true);
+                    item.EquipmentSlot = slot.ToUpper();
```
Comment (AlanMorel) [inline Ms2Database/Controllers/EquipmentManager.cs:49]: Seems like you can move this and line 44 to after the if/else, then maybe invert line 42 to call line 48

**Review by AlanMorel — CHANGES_REQUESTED** [review]

File: Ms2Database/Controllers/EquipmentManager.cs (hunk 1 above)
Comment (bwu113) [reply Ms2Database/Controllers/EquipmentManager.cs]: Gone ahead with the suggestion.

**Comment by AlanMorel** [comment]
Looks good after the merge conflict is fixed

**Review by angelotadeucci — APPROVED** [review]
Looks good. Also fix your git history

**Pull request merged**